src/modules/Auth/index.js

```diff
@@ -9,17 +9,21 @@ class Auth extends Component {
   }
 
   startAuthorization = () => {
-    // https://api.imgur.com/oauth2/authorize?client_id=YOUR_CLIENT_ID&response_type=REQUESTED_RESPONSE_TYPE&state=APPLICATION_STATE
-    alert("Start Authorization");
+    const authorizationUrl = `https://api.imgur.com/oauth2/authorize?client_id=${
+      process.env.CLIENT_ID
+    }&response_type=token`;
+
+    console.log(authorizationUrl);
```

Comment:
remove console.log

Reply:
Will not push any `console.log` from now on.